tests/unit/ResultsContainer.spec.js

```diff
@@ -0,0 +1,90 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+import ResultsContainer from '@/views/ResultsContainer.vue'
+import ResultsList from '@/components/ResultsList.vue'
+import ResultsMap from '@/components/ResultsMap.vue'
+import VueRouter from 'vue-router'
+import Vuex from 'vuex'
+
+const localVue = createLocalVue()
+localVue.use(VueRouter)
+localVue.use(Vuex)
+const router = new VueRouter()
+
+describe('ResultsContainer happy paths', () => {
```

Comment:
Enjoy the labeling for will be happy path and what will be sad path!

Reply:
I set that up mainly because I realized I needed two separate sets of data to mount the components.  For example, for sad path, I might have an empty array for `searchResults`, but for happy path, I need results to work with.  Having two separate `describe` blocks made that easy and kept the unit tests relatively clean.